.ci/test_r_package.sh

```diff
@@ -90,6 +90,24 @@ if [[ $OS_NAME == "macos" ]]; then
 fi
 Rscript --vanilla -e "install.packages(${packages}, repos = '${CRAN_MIRROR}', lib = '${R_LIB_PATH}', dependencies = c('Depends', 'Imports', 'LinkingTo'))" || exit -1
 
+if [[ $TASK == "check-r-docs" ]]; then
+    Rscript build_r.R
```

Comment:
Maybe for fast-fail?
```suggestion
    Rscript build_r.R || exit -1
```